sirepo/job.py

```diff
@@ -177,27 +166,13 @@ def init():
             str,
             'shared secret between supervisor and server',
         ),
-        supervisor_uri=(
-            'http://{}:{}'.format(DEFAULT_IP, DEFAULT_PORT),
```

Comment:
This should be the default for dev. I think we use this value statically (DEFAULT_SUPERVISOR_URI) for the default case for all the specific cfg.supervisor_uri values.